tests/devices/test_default_qubit_tf.py

```diff
@@ -927,6 +928,7 @@ def test_three_qubit_no_parameters_broadcasted(self, broadcasted_init_state, op,
         expected = np.einsum("ij,lj->li", mat, state)
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
+    @pytest.mark.usefixtures("use_new_opmath")
```

Comment:
Perhaps we can use `qml.ops.LinearCombination` in the test explicitly. Currently this reads a bit confusingly when not scrolling down to see the other test and knowing how we changed the tests in the new opmath initiative

Reply:
done